test/fixtures/test-runner/test_only.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+const test = require('node:test');
+
+test('this should be skipped');
+test.only('this should be executed');
```

Comment:
add a test also for `{ only: true }` and `{ only: false }`?
```suggestion
test('this should be skipped', { only: false });
test.only('this should be executed');
test('this should be executed', { only: true });
```

Reply:
that is tested extensively in https://github.com/nodejs/node/blob/main/test/fixtures/test-runner/output/only_tests.js